spec/config/memoized_value_spec.rb

```diff
@@ -2,10 +2,20 @@
 
 RSpec.describe Pry::Config::MemoizedValue do
   describe "#call" do
-    subject { described_class.new { rand } }
-
     it "memoizes the result of call" do
-      expect(subject.call).to eq(subject.call)
+      instance = described_class.new { rand }
+      expect(instance.call).to eq(instance.call)
+    end
+
+    specify "even if the result is falsy" do
```

Comment:
This probably works not the way you expect. With randomised specs the "even if the result is false" part will look a little out of place:

```
% bundle exec rspec --format=doc spec/config/memoized_value_spec.rb --order=random

Randomized with seed 65243

Pry::Config::MemoizedValue
  #call
    even if the result is falsy
    memoizes the result of call
```

What if we reshuffled this with `context`:

```
Pry::Config::MemoizedValue
  #call
    when result is nil
       memoizes the result of call
    when result is not nil
       memoizes the result of call
```

Or in any other sensible way that reads better, I don't have a preference. It's just that the "even if the result is falsy" will be hard to make sense of if we add more tests to for this method and randomise them.